cli/src/main/java/com/box/l10n/mojito/cli/pseudoloc/PseudoLocalization.java

```diff
@@ -0,0 +1,193 @@
+package com.box.l10n.mojito.cli.pseudoloc;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author srizvi
+ */
+public class PseudoLocalization {
+
+	private static Map<String, String> pseudoLocMap = new HashMap<String, String>();
+	static {
+		pseudoLocMap.put("a", "àáâãäåāăąǻάαа");
+		pseudoLocMap.put("A", "ÀÁÂÃÄÅĀĂĄǺΆΑА");
+		pseudoLocMap.put("b", "вьъ");
+		pseudoLocMap.put("B", "ΒßβБВ");
+		pseudoLocMap.put("c", "¢çćĉċčсς");
+		pseudoLocMap.put("C", "ÇĆĈĊČС");
+		pseudoLocMap.put("d", "ďđ");
+		pseudoLocMap.put("D", "ÐĎĐ");
+		pseudoLocMap.put("e", "èéêëēĕėęěέεеёє℮");
+		pseudoLocMap.put("E", "ÈÉÊËĒĔĖĘĚΈΕΣЕЁЄЄ");
+		pseudoLocMap.put("f", "ƒ");
+		pseudoLocMap.put("F", "₣");
+		pseudoLocMap.put("g", "ĝğġģ");
+		pseudoLocMap.put("G", "ĜĞĠĢ");
+		pseudoLocMap.put("h", "ĥħнћ");
+		pseudoLocMap.put("H", "ĤĦΉΗН");
+		pseudoLocMap.put("i", "ìíîïĩīĭįίιϊіїΐ");
+		pseudoLocMap.put("I", "ÌÍÎĨĪĬĮİΊΪІЇ");
+		pseudoLocMap.put("j", "ĵј");
+		pseudoLocMap.put("J", "ĴЈ");
+		pseudoLocMap.put("k", "ķĸκкќ");
+		pseudoLocMap.put("K", "ĶΚЌК");
+		pseudoLocMap.put("l", "ĺļľł");
+		pseudoLocMap.put("L", "ĹĻĽĿŁ");
+		pseudoLocMap.put("m", "mм");
+		pseudoLocMap.put("M", "ΜМм");
+		pseudoLocMap.put("n", "ийлпπήηńņňŉŋñ");
+		pseudoLocMap.put("N", "ÑŃŅŇŊΝИЙП");
+		pseudoLocMap.put("o", "òóôõöøōŏőοσόоǿ");
+		pseudoLocMap.put("O", "ÒÓÔÕÖØŌŎŐǾΌΘΟО");
+		pseudoLocMap.put("p", "ρр");
+		pseudoLocMap.put("p", "ΡР");
+		pseudoLocMap.put("r", "ŕŗřяѓґгř");
+		pseudoLocMap.put("R", "ŔŖŘЯΓЃҐГ");
+		pseudoLocMap.put("s", "śŝşѕš");
+		pseudoLocMap.put("S", "ŚŜŞЅŠ");
+		pseudoLocMap.put("t", "ţťŧτт");
+		pseudoLocMap.put("T", "ŢŤŦΤТ");
+		pseudoLocMap.put("u", "µùúûüũūŭůűųцμџ");
+		pseudoLocMap.put("U", "ÙÚÛÜŨŪŬŮŰŲЏЦ");
+		pseudoLocMap.put("v", "ν");
+		pseudoLocMap.put("w", "ŵωώшщẁẃẅ");
+		pseudoLocMap.put("W", "ŴШЩẀẂẄ");
+		pseudoLocMap.put("x", "×хж");
+		pseudoLocMap.put("X", "ΧχХЖ");
+		pseudoLocMap.put("y", "ýÿŷγУўỳу");
+		pseudoLocMap.put("Y", "ÝŶΎΥΫỲЎ");
+		pseudoLocMap.put("z", "źżž");
+		pseudoLocMap.put("Z", "ŹŻΖŽ");
+	}
+
+	/**
+	 * Pseudo localize a string depending on instance settings to choose the type, expansion factor and bracket option.
+	 * If pseudo-localization is disabled, returns the passed in string
+	 *
+	 * @param string string to be pseudo localized
+	 * @return pseudo localized string
+	 */
+	public static String convertStringToPseudoLoc(String string)
```

Comment:
better to not use static methods. In the rest of the code we use DI and class instances. It makes things easier to test and mock, etc.